testsupport/space.go

```diff
@@ -20,14 +19,14 @@ import (
 var notAllowedChars = regexp.MustCompile("[^-a-z0-9]")
 
 // NewSpace initializes a new Space object with the given options. By default, it doesn't set anything in the spec.
-func NewSpace(awaitilities wait.Awaitilities, opts ...SpaceOption) *toolchainv1alpha1.Space {
-	namePrefix := strings.ToLower(awaitilities.Host().T.Name())
+func NewSpace(t *testing.T, awaitilities wait.Awaitilities, opts ...SpaceOption) *toolchainv1alpha1.Space {
+	namePrefix := strings.ToLower(t.Name())
 	// Remove all invalid characters
 	namePrefix = notAllowedChars.ReplaceAllString(namePrefix, "")
 
-	// Trim if the length exceeds 50 chars (63 is the max)
-	if len(namePrefix) > 50 {
-		namePrefix = namePrefix[0:50]
+	// Trim if the length exceeds 40 chars (63 is the max)
+	if len(namePrefix) > 40 {
+		namePrefix = namePrefix[0:40]
```

Comment:
just out of curiosity - why do we need to make them shorter?

Reply:
IIRC, I ran into errors because `t` is now the name of the subtest (hence, it is longer), and the whole name would be longer than 63 characters

Reply:
ok :+1: